modules/core/arrow-core-extensions/src/main/kotlin/arrow/core/extensions/ior.kt

```diff
@@ -96,6 +98,26 @@ interface IorTraverse<L> : Traverse<IorPartialOf<L>>, IorFoldable<L> {
     fix().traverse(AP, f)
 }
 
+@extension
+interface IorBifoldable : Bifoldable<ForIor> {
+  override fun <A, B, C> IorOf<A, B>.bifoldLeft(c: C, f: (C, A) -> C, g: (C, B) -> C): C =
+    fix().fold({ f(c, it) }, { g(c, it) }, { a, b -> g(f(c, a), b) })
```

Comment:
Same here, move all instances implementations when possible to the data type. Future versions of arrow won't export the syntax as individual extension functions and we are interested in the data type always including the relevant methods that it wants to expose directly without the type class indirection were possible.